Sources/Views/Collections/CollectionSection.swift

```diff
@@ -187,7 +187,9 @@ private func relatedItem(_ relatedItem: Episode.Collection.Section.Related) -> N
   return .gridColumn(
     sizes: [.mobile: 12],
     attributes: [
-      .style(margin(top: .px(4))),
+    .class([
+      Class.padding([.mobile: [.bottom: 2]])
```

Comment:
it didn't see the `4px` margin was necessary, but I could have missed something. I added some extra bottom padding so that the blurb+episode for each related item was a little separated from the next one